core/src/ldml/C7043_ldml.md

```diff
@@ -324,10 +324,120 @@ For each key:
 - `vkey`: Is the standard vkey, 0-255
 - `target`: Is the target (resolved) vkey, 0-255.
 
+### C7043.2.x `lyrs`—Layers list
+
+| ∆ | Bits | Name    | Description                              |
+|---|------|---------|------------------------------------------|
+| 0 |  32  | ident   | `lyrs`                                   |
+| 4 |  32  | size    | int: Length of section                   |
+| 8 |  32  | count   | int: Number of layerses                  |
+|12 |  32  | reserved| padding                                  |
+
+For each 'layers':
+
+| ∆ | Bits | Name       | Description                              |
+|---|------|------------|------------------------------------------|
+|16+|  32  | flags      | int: per-layers options                  |
+|24+|  32  | hardware   | str: layout (`us`,`iso`,`jis`,`abnt2`)   |
+|20+|  32  | target     | int: target vkey ID (0…255)              |
+|28+|  32  | layer      | int: `layr` identifier of first layer    |
+|32+|  32  | layerCount | int: number of `layr` elements           |
+|36+|  96  | reserved   | short`[3]`: padding                      |
+
+- `flags`: a 32-bit bitfield defined as below:
+
+  | Bit position | Meaning  |  Description         |
+  |--------------|----------|----------------------|
+  |       0      | form     | 0: hardware          |
+  |       0      | form     | 1: touch             |
+
+### C7043.2.x `layr`—Layer list
```

Comment:
I think I'd be inclined to fold this into the previous section

Reply:
sure i can see that. just add 1 offset, 'start of layer list'

Reply:
ptal